src/grt/src/GlobalRouter.cpp

```diff
@@ -1880,6 +1882,56 @@ void GlobalRouter::connectPadPins(NetRouteMap& routes)
   }
 }
 
+void GlobalRouter::computePinOffsetAdjustments()
+{
+  for (auto net : block_->getNets()) {
```

Comment:
I think you don't need to iterate over all nets. Just iterate directly into the `pad_pins_connections_` map. A reference here: https://stackoverflow.com/questions/26281979/c-loop-through-map

Reply:
Yes I wanted to improve that but I had never iterated over a map before and forgot to do it, thanks!